nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -1,3 +1,11 @@
+- project:
+    name: nfs-ganesha
+    scenario:
```

Comment:
This doesn't need to be called ``scenario``, it could just be ``ceph-branch`` or something that identifies what this value better.